CHANGELOG.md

```diff
@@ -4,7 +4,16 @@
 
 ### Google Artifact Registry Support
 Cromwell now supports call caching when using Docker images hosted on
-[Google Artifact Registry](https://cloud.google.com/artifact-registry). 
+[Google Artifact Registry](https://cloud.google.com/artifact-registry).
+
+### Google Image Repository Hashing Updates
```

Comment:
I love this! It also really helps me as a reviewer to see this at the top because now I know what to expect from the rest of the PR.